onnxruntime/test/testdata/transform/fusion/isinf_reducesum.py

```diff
@@ -0,0 +1,53 @@
+import onnx
+from onnx import helper
+from onnx import TensorProto, OperatorSetIdProto
+from enum import Enum
+
+opsets = []
+onnxdomain = OperatorSetIdProto()
+onnxdomain.version = 12
+onnxdomain.domain = "" # The empty string ("") or absence of this field implies the operator set that is defined as part of the ONNX specification.
+opsets.append(onnxdomain)
+
+msdomain = OperatorSetIdProto()
+msdomain.version = 1
+msdomain.domain = 'com.microsoft'
+
+opsets.append(msdomain)
+kwargs={}
+kwargs['opset_imports'] = opsets
+
+def GenerateModel(model_name):
+    nodes = [  # subgraph
+        helper.make_node("Cast", ["A"], ["cast1"], "cast_1", to=11),
+
+        helper.make_node("IsInf", ["cast1"], ["IsInf_out"], "is_inf"),
+
+        helper.make_node("Cast", ["IsInf_out"], ["cast2"], "cast_2", to=7),
```

Comment:
would these two casts always be there?

Reply:
saw your comments in the code

---
In reply to: [604599029](https://github.com/microsoft/onnxruntime/pull/7188#discussion_r604599029) [](ancestors = 604599029)